docs/UserGuide.md

```diff
@@ -135,22 +138,31 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd student in the application.
 * `find Betsy` followed by `delete 1` deletes the 1st student in the results of the `find` command.
 
-#### Clearing all entries : `clear`
+#### Clearing all students : `clear`
 
-Clears all entries from the application.
+Clears all students from the application.
 
 Format: `clear`
 
-### Managing Classes: [Coming soon...]
+### Managing Classes: [coming soon]
 
 #### Adding a class: `add-class`
 
 Adds a class to the application.
 
-Format: `add-class n/NAME`
+Format: `add-class [n/CLASS_NAME] [m/MODULE_CODE]`
 
 Examples:
-* `add-class n/CS2013T Tutorial T10`
+* `add-class n/Tutorial T10 m/CS2013T`
+
+#### Listing all students within class : `list students`
+
+Shows a list of all students within a particular class in the application.
+
+Format: `list students /by [n/CLASS_NAME] [m/MODULE_CODE]`
```

Comment:
We need to rethink the command syntax for this operation. The parsing of arguments will be troublesome in this format.

For now maybe we can leave this feature out of the UG.

Reply:
Resolved by removing MODULE_CODE